client/components/checklist/style.scss

```diff
@@ -2,20 +2,38 @@
 	display: flex;
 	flex-direction: row;
 
+	@include breakpoint( '<480px' ) {
+		flex-direction: column;
+		justify-content: flex-end;
+	}
+
+	&.card {
+		padding: 0;
```

Comment:
This removed the need of `position: absolute for `&-action` (line 50 below) and makes applying regular Flexbox rules much easier.

The only caveat is that we won't inherit paddings from the card and we'll need to apply them ourself. I wish we'd have variables for those to ensure everything stays consistent, but that's another story...